### ISSUE TITLE
XML feeds removed from NFL website?

### ISSUE THREAD

**Issue by danielcroona** [BurntSushi/nfldb#320]
In `live.py` there is a reference to the schedule in XML format on the following URL: `http://www.nfl.com/liveupdate/scorestrip/ss.xml`

Now when I try to update the schedule there is a XML issue, apparently because the XML is removed from the NFL website. Does anyone know if it is available on another URL? Is it possible to use data from [NFL API](https://api.nfl.com/docs/league/models/game/index.html) instead?

I know this project is no longer maintained but if someone found a solution to this issue, please let me know.

**Comment by danielcroona** [comment]
Update: apparently the data is still there, but on another domain:
http://static.nfl.com/liveupdate/scorestrip/ss.xml

**Comment by CPAllen55** [comment]
Hey, so all you did was change out the url from the live.py file? Did you have any other issues?

**Comment by danielcroona** [comment]
Yes, there was. I used this primarily for getting info about rosters and that didn't work at all so I had to give up on the whole thing unfortunately.

**Comment by Edorn71** [comment]
Unfortunately you can only use this to download historical Play by Play data now, the NFL appears to have ceased publishing the Play by Play statistics live.  If all you want to do is analyse complete seasons, this does appear to work, but until next season starts it won't be clear if 2020 will start being made available.
